tests/telemetry/telemetry_test.go

```diff
@@ -53,12 +50,15 @@ var _ = Describe("{DiagsBasic}", func() {
 					OnHost:        true,
 				}
 				err := Inst().V.CollectDiags(currNode, config, torpedovolume.DiagOps{Validate: true})
+				Step(fmt.Sprintf("error is: %s", err), func() {
```

Comment:
do we really need a step to print the error here?

Reply:
nope removing